generative_ai/prompts/prompt_template.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def prompt_template() -> str:
+    """Create prompt template"""
+
+    # [START generativeaionvertexai_prompt_template]
+    import vertexai
+    from vertexai.preview.prompts import Prompt
+
+    # Initialize vertexai
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    variables = [
+        {
+            "animal": ["""Eagles, Coyotes, Squirrels"""],
```

Comment:
Do you mean to only pass one animal and activity at a time? If so, these variables are incorrectly formatted.

Here's how to correctly format them
variables = [
{"animal": "Eagles", "activity":  "eat berries"}, {"animal": "Coyotes", "activity": "jump"}, {"animal": "Squirrels", "activity": "fly"}
]